tox.ini

```diff
@@ -189,8 +189,8 @@ commands =
 description = Update requirements.txt files
 skip_install = true
 deps =
-    pip
-    pip-tools
+    pip==20.3.3
```

Comment:
The pip 21 is available. It could be interesting to use it  ?

Reply:
Without the fixed versions, I found a combo of pip/piptools that did not work together in the tox pip-compile env, so it fixed the version so that the pip-compile command works. There is not interest here to fix something here IMHO.